Rebuild-Black-Business/RBB-Website
# fixed sidenav scroll issue

This PR fixes the mobile nav scrolling issue, whereby the user could scroll the page when the nav is open.

Ive left some comments in the code so its clear whats going on. Not super complicated or anything, just so people know

## Pages/Interfaces that will change

All pages

### Screenshots / video of changes

Couldnt really take a screenshot of this behavior 

## Steps to test

Local dev, dev tools, mobile view. Then click the nav, it should lock in place, navigate to a new page and you should be able to scroll. Rinse and repeat
